<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Host:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEditHost">
          <property name="text">
           <string>127.0.0.1</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Port:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="spinBoxPort">
          <property name="maximum">
           <number>99999</number>
          </property>
          <property name="value">
           <number>1883</number>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>IP:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="lineEdit_8">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="buttonConnect">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="5" column="2">
       <widget class="QLabel" name="labelMsg_4">
        <property name="text">
         <string>Message 4:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="4">
       <widget class="QLineEdit" name="lineEditMessage_6"/>
      </item>
      <item row="10" column="4">
       <widget class="QLineEdit" name="lineEditMessage_9"/>
      </item>
      <item row="11" column="5">
       <widget class="QPushButton" name="buttonPing">
        <property name="text">
         <string>Ping</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="labelMsg_3">
        <property name="text">
         <string>Message 3:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditTopic_2"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelTopic_2">
        <property name="text">
         <string>Topic 2:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="5">
       <widget class="QPushButton" name="buttonPublish_4">
        <property name="text">
         <string>Publish</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="4">
       <widget class="QLineEdit" name="lineEditTopic">
        <property name="text">
         <string>ASI1/MAIN/#</string>
        </property>
       </widget>
      </item>
      <item row="6" column="4">
       <widget class="QLineEdit" name="lineEditMessage_5"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditTopic_1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>ASI1/AGV/100/SI3</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelTopic_3">
        <property name="text">
         <string>Topic 3:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Message 6:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="labelTopic_5">
        <property name="text">
         <string>Topic 5:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEditTopic_3"/>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Topic 8:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="lineEditTopic_7"/>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="labelMsg_2">
        <property name="text">
         <string>Message 2:</string>
        </property>
       </widget>
      </item>
      <item row="9" column="4">
       <widget class="QLineEdit" name="lineEditMessage_8"/>
      </item>
      <item row="10" column="1">
       <widget class="QLineEdit" name="lineEditTopic_9"/>
      </item>
      <item row="9" column="5">
       <widget class="QPushButton" name="buttonPublish_8">
        <property name="text">
         <string>Publish</string>
        </property>
       </widget>
      </item>
      <item row="10" column="5">
       <widget class="QPushButton" name="buttonPublish_9">
        <property name="text">
         <string>Publish</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEditTopic_4"/>
      </item>
      <item row="10" column="2">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Message 9:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelTopic_1">
        <property name="text">
         <string>Topic 1:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="buttonSubscribe">
        <property name="text">
         <string>Subscribe</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLineEdit" name="lineEditMessage_1">
        <property name="text">
         <string>101,19,2704,1,0,0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="labelMsg_1">
        <property name="text">
         <string>Message 1:</string>
        </property>
       </widget>
      </item>
      <item row="9" column="2">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Message 8:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="4">
       <widget class="QLineEdit" name="lineEditMessage_7"/>
      </item>
      <item row="8" column="2">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Message 7:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="lineEditTopic_6"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelTopicMain">
        <property name="text">
         <string>Topic:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="5">
       <widget class="QPushButton" name="buttonPublish_6">
        <property name="text">
         <string>Publish</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLineEdit" name="lineEditMessage_2"/>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="lineEditTopic_5"/>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Topic 9:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLabel" name="labelMsg_5">
        <property name="text">
         <string>Message 5:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QLineEdit" name="lineEditMessage_3"/>
      </item>
      <item row="9" column="1">
       <widget class="QLineEdit" name="lineEditTopic_8"/>
      </item>
      <item row="8" column="5">
       <widget class="QPushButton" name="buttonPublish_7">
        <property name="text">
         <string>Publish</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="labelTopic_4">
        <property name="text">
         <string>Topic 4:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="4">
       <widget class="QLineEdit" name="lineEditMessage_4"/>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Topic 7:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Topic 6:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QPushButton" name="buttonPublish_1">
        <property name="text">
         <string>Publish</string>
        </property>
       </widget>
      </item>
      <item row="6" column="5">
       <widget class="QPushButton" name="buttonPublish_5">
        <property name="text">
         <string>Publish</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QPushButton" name="buttonPublish_2">
        <property name="text">
         <string>Publish</string>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="QPushButton" name="buttonPublish_3">
        <property name="text">
         <string>Publish</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Log Messages</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPlainTextEdit" name="editLog"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="buttonQuit">
        <property name="text">
         <string>Quit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionDoc"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionDoc">
   <property name="text">
    <string>Doc</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
